web/src/app/models/userSocial.model.ts

```diff
@@ -1,8 +1,28 @@
 import { Social } from './social.model';
 
 export interface UserSocial {
-  github?: Social;
-  twitter?: Social;
-  youtube?: Social;
-  instagram?: Social;
+  github?: {
+    updatedAt: string,
+    followers: number,
+    following: number,
+    userId: string,
```

Comment:
if `userId` was promoted to be a sibling to the providers then it won't need to be duplicated for each social. Also could use the `Social` model.

```typescript
{
userId: string;
  github?: Social;
  twitter?: Social;
  // ...
}
```